docs/source/complex_numbers.rst

```diff
@@ -0,0 +1,188 @@
+.. _complex_numbers-doc:
+
+Complex Numbers
+===============
+
+Complex Numbers are numbers that can be expressed in the form :math:`a + bi`, where a and b are real numbers,
+and *i* is a solution of the equation :math:`x^2 = −1`. Complex numbers frequently occur in mathematics and
+engineering, especially in signal processing. The aim of introducing tensors of complex dtypes is to provide
+a more natural user experience for users and libraries (eg. TorchAudio) that currently work around the
+lack of complex tensors by using Float Tensors with shape :math:`(..., 2)` where the last dimension contains
```

Comment:
This seems like it's underselling the feature. The goal is to not just replace existing workarounds, right?

Reply:
I would consider introducing these existing workarounds in its own section later. Maybe a section like: "adapting existing code"?